common/predictive-text/testing/two-stage-embedded-webworker/main.ts

```diff
@@ -0,0 +1,30 @@
+// Provides the final source for our compiled WebWorker within a wrapping function.
+/// <reference path="embedded_worker.js" />
+
+function createWorker(fn: Function): Worker {
+  var str_fn = fn.toString();
+
+  // We now unwrap our WebWorker from its function.
+  var str_lines = str_fn.split("\n");
+  var blob_lines = []
+
+  for(var i=1; i < str_lines.length -1; i++) {
+    blob_lines.push(str_lines[i] + "\n");
+  }
+  // Unwrapping complete.
```

Comment:
I just noticed this today, and I spent a bit of time figuring out _why_ it was there. I understand why (I think), but it was a bit of a head-scratchy moment. My concern does not require changes in `testing/`, but when I'm writing code in `predictive-text/`, I'd like @jahorton to please confirm the purpose of this "unwrapping":

So, I gather that, because the embedded worker looks like this:

```javascript
var LMLayerWorker = function() {
var counter = 0;
onmessage = function (e) {
    counter++;
    console.log("Message received from main page: ", e.data);
    postMessage(counter);
};
//# sourceMappingURL=worker.js.map
}
```

And because the stuff inside `LMLayerWorker` is _supposed to be in the Worker's global scope, then the "unwrapping" extracts code out of the generated function wrapper `LMLayerWorker`'s syntax (i.e., the first line and the last line).

Therefore, the blob becomes this:

```javascript
var LMLayerWorker = function() {
var counter = 0;
onmessage = function (e) {
    counter++;
    console.log("Message received from main page: ", e.data);
    postMessage(counter);
};
//# sourceMappingURL=worker.js.map
}
```

This is what the unwrapping is for, correct?

Again, no changes need to be made here, but in the non-testing code, I'll make this a bit more clear.